routers/api/v1/repo/issue.go

```diff
@@ -251,6 +256,33 @@ func ListIssues(ctx *context.APIContext) {
 		}
 	}
 
+	var mileIDs []int64
+	if part := strings.Split(ctx.Query("milestones"), ","); len(part) > 0 {
+		for i := range part {
+			mile, err := models.GetMilestoneByRepoIDANDName(ctx.Repo.Repository.ID, part[i])
+			if err != nil {
```

Comment:
imho this could should be rearranged to be more readable:

```go
			if err == nil {
				mileIDs = append(mileIDs, mile.ID)
				continue
			}
			if !models.IsErrMilestoneNotExist(err) {
				ctx.Error(http.StatusInternalServerError, "GetMilestoneByRepoIDANDName", err)
				return
			}
			id, err := strconv.ParseInt(part[i], 10, 64)
			if err != nil {
				continue
			}
			mile, err = models.GetMilestoneByRepoID(ctx.Repo.Repository.ID, id)
			if err == nil {
				mileIDs = append(mileIDs, mile.ID)
				continue
			}
			if models.IsErrMilestoneNotExist(err) {
				continue
			}
			ctx.Error(http.StatusInternalServerError, "GetMilestoneByRepoID", err)
```

Reply:
done :)